lib/worker.js

```diff
@@ -58,21 +60,15 @@ function shim(Linter) {
   return LinterShim;
 }
 
-function loadDefaultTSLint() {
-  if (!tslintDef) {
-    // eslint-disable-next-line import/no-dynamic-require
-    tslintDef = require(tslintModuleName).Linter;
-  }
-}
-
-async function getLocalLinter(basedir) {
+function resolveAndCacheLinter(fileDir, moduleDir) {
+  const baseDir = moduleDir || fileDir;
   return new Promise((resolve) => {
     if (!requireResolve) {
       requireResolve = require('resolve');
     }
     requireResolve(
       tslintModuleName,
-      { basedir },
+      { basedir: baseDir },
```

Comment:
Looks like this got lost in the mess of comments, since this name needs to be sent as an object into `requireResolve`, let's just keep it as `basedir` in the entire function. Sorry for going back and forth on this!

Reply:
woops! It's okay, I'll change it back. haha